lib/carrierwave/storage/file.rb

```diff
@@ -28,11 +28,8 @@ class File < Abstract
       #
       def store!(file)
         path = ::File.expand_path(uploader.store_path, uploader.root)
-        if uploader.move_to_store
-          file.move_to(path, uploader.permissions, uploader.directory_permissions)
-        else
-          file.copy_to(path, uploader.permissions, uploader.directory_permissions)
-        end
+        strategy = uploader.move_to_store ? 'move_to' : 'copy_to'
+        file.send(strategy, path, uploader.permissions, uploader.directory_permissions)
```

Comment:
I don't like calling `send` pointlessly. It skews the Ruby backtrace and we are going to have a bad time during debugging stuff.